### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ndv20**
[DEMO LINK](https://dmytronykorovych.github.io/react_tabs-js/)

### CODE REVIEW

File: src/components/Tabs/Tabs.jsx [hunk 1]
```diff
@@ -1 +1,40 @@
...
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
```
Comment (maxim2310) [inline src/components/Tabs/Tabs.jsx:11]:
```suggestion
          {tabs.map(({ id, title }) => {
```

File: src/components/Tabs/Tabs.jsx [hunk 2]
```diff
@@ -1 +1,40 @@
...
+                key={id}
+                className={cn({ 'is-active': isActive })}
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => (
+                    !isActive && onTabSelected(tab)
+                  )}
```
Comment (maxim2310) [inline src/components/Tabs/Tabs.jsx:25]: need to move to separate func

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Good work, but lets make some improvement

**Review by anastasiia-tilikina — APPROVED** [review]
GJ

**Pull request closed without merging**